Testing/SystemTests/tests/analysis/SaveNexusTest.py

```diff
@@ -0,0 +1,124 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#pylint: disable=no-init,invalid-name,too-few-public-methods
+from __future__ import (absolute_import, division, print_function)
+from mantid.simpleapi import *
+from mantid.api import FrameworkManager
+import os
+import glob
+
+EXPECTED_EXT = '.expected'
+
+# files blacklisted due to containing duplicate named components in instrument.
+file_blacklist = [("HET_Definition_old.xml","duplicate monitor names"), ("MAPS_Definition.xml","duplicate bank names"),
```

Comment:
Good. Rather than copy-paste the messages (since they tend to repeat) could you define them somewhere. Or another approach do something like this...
```python
duplicate_monitor_names = [file1.xml, file2.xml, file5.xml] 
duplicate_bank_names = [file1.xml, file2.xml, file5.xml] 
black_list = duplicate_monitor_names + duplicate_bank_names
```

Reply:
done